### PR TITLE
Add .head function

### PR DESCRIPTION
**Pull request by noffle**
Addresses #111.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -781,6 +781,12 @@ Feed.prototype.has = function (start, end) {
   return total === this.bitfield.total(start, end)
 }
 
+Feed.prototype.head = function (opts, cb) {
+  if (typeof opts === 'function') return this.head({}, opts)
+  if (this.length === 0) cb(new Error('feed is empty'))
+  else this.get(this.length - 1, cb)
```
Comment (mafintosh) [inline index.js]: you should wrap this in .ready so .length is always loaded :) can we make it return null, null if there is no head?

Comment (mafintosh) [inline index.js]: nevermind the null, null - i guess we return an error in .get as not found as well

Comment (noffle) [inline index.js]: Do you think it would be confusing to have `.get` return immediately but `.head` to wait for `ready`?

Comment (mafintosh) [inline index.js]: .get waits for ready right now

Comment (noffle) [inline index.js]: Understood :bulb:

**Comment by mafintosh** [comment]
6.7.0

**Pull request merged**